gnucash/report/standard-reports/transaction.scm

```diff
@@ -1487,9 +1482,10 @@ Credit Card, and Income accounts."))))))
             "<b>Split Reconciled Status: </b>"
             (string-join
              (map (lambda (item)
-                    (cdr (assq item (list (cons #\n "unreconciled")
-                                          (cons #\c "cleared")
-                                          (cons #\y "reconciled")))))
+                    (case item
+                      ((#\n) "unreconciled")
+                      ((#\c) "cleared")
+                      ((#\y) "reconciled")))
```

Comment:
Can we reuse the strings as used in the combobox (in this case that means capitalized) ?
That would make for less translatable strings (that is they can be reused more that once).